NuimoSDK/Properties/AssemblyInfo.cs

```diff
@@ -1,18 +1,20 @@
 ﻿using System.Resources;
 using System.Reflection;
+using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
 // General Information about an assembly is controlled through the following 
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
-[assembly: AssemblyTitle("Nuimo SDK")]
-[assembly: AssemblyDescription("Library for Windows 10 platforms to connect and communicate with Nuimo controllers made by Senic")]
+[assembly: AssemblyTitle("NuimoSDK")]
+[assembly: AssemblyDescription("")]
 [assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("Senic GmbH")]
-[assembly: AssemblyProduct("Nuimo SDK")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("NuimoSDK")]
```

Comment:
My mistake. I made this PR by creating a whole new project and then include the source files. I forgot to overwrite the default AssemblyInfo.cs with yours.